editor/src/core/model/element-metadata-utils.ts

```diff
@@ -500,6 +500,13 @@ export const MetadataUtils = {
     }
     return result
   },
+  getDescendantPaths(
+    elements: ElementInstanceMetadataMap,
+    target: ElementPath,
+  ): Array<ElementPath> {
+    const allPaths = Object.keys(elements).map(EP.fromString)
```

Comment:
If I got this right it'll save on the need to call `EP.fromString`:
```suggestion
    const allPaths = Object.values(elements).map(elem => elem.elementPath)
```

Reply:
Good shout! We could go even further and combine with the next line using a `mapDropNulls` too, so I'm going to do that as well

Reply:
I've added a new function `mapAndFilter` since we use this pattern a lot